cmd/agent/flow.go

```diff
@@ -43,6 +43,7 @@ func runFlow() error {
 		httpListenAddr = "127.0.0.1:12345"
 		configFile     string
 		storagePath    = "data-agent/"
+		ready          = true
```

Comment:
Wouldn't this be false by default?

Reply:
The http listener is only available once the config has been loaded successfully. We could start as false but would have to add an additional line of code to flip it to true after reload().